lib/node_modules/@stdlib/math/base/special/hyp2f1/lib/main.js

```diff
@@ -0,0 +1,284 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+var gamma = require( '@stdlib/math/base/special/gamma' );
+var round = require( '@stdlib/math/base/special/round' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var PINF = require( '@stdlib/constants/math/float64-pinf' );
+var max = require( '@stdlib/math/base/special/max' );
+
+var { hys2f1 } = require( './support.js' );
+var { hyt2f1 } = require( './support.js' );
```

Comment:
```suggestion
var hyt2f1 = require( './support.js' ).hyt2f1;
```
As we target ES5 environments, project convention is not to use ES2015+ features, such as destructuring.